nebari/stages/input_vars.py

```diff
@@ -64,6 +64,7 @@ def stage_02_infrastructure(stage_outputs, config):
             "environment": config["namespace"],
             "region": config["google_cloud_platform"]["region"],
             "project_id": config["google_cloud_platform"]["project"],
+            "kubernetes_version": config["google_cloud_platform"]["kubernetes_version"],
```

Comment:
I was going to point this out, good catch. Are there any other inputs missing for the different cloud providers?

Reply:
I haven't noticed any other ones missing but I will keep my eye out for them. I issue I am currently running into is how to set the kubernetes version from the [Google Terraform provider](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster) 😅 I thought I could set the `master_version` but turns out that is a read-only attribute. Still investigating.